content/blog/standup.md

```diff
@@ -0,0 +1,212 @@
+
+##### Q: Where?
+
+Anywhere the whole team is comfortable. Physical or Remote. Does not matter.
+
+If you are conducting it remotely, always have videos on.
+You want to talk to humans, not animated figures!
+
+##### Q: When?
+
+Ideally at the start of the day. However, if everyone is not available or working remotely in different geographies, you
+can do it anytime.
+
+##### Q: What content?
+
+- What did I do since the last meeting?
+- What will I do until the next meeting?
+- Challenges
+
+##### Q: How granular?
+
+Remember, it is for the team. Your updates should be abstract enough that everyone can understand and contribute to the project goal.
+
+##### Q: Duration?
+
+Ideally, 15 minutes max. Every team member should speak for only 2 or 3 minutes. If there are too many
+participants and not everyone can finish in 15 minutes, then it is a different problem. Ideally, divide the
+team instead of shortening individuals' time or increasing standup meetings' time.
+
+##### Q: Should I be standing up?
+
+Well, the meeting should be short. So, psychologically, if you are standing up, you would be quick. So, if your standups
+are not ending in time, you should definitely start standing up.
+
+##### Q: Should we hire a scrum master?
+
+If you are married to the process of Scrum, then sure. But, if your focus is on Agile values, you don't need to.
+
+I disagree with the concept of having a Scrum master. The word master is terrible anyway. 
+It also brings all the micromanagement, like tracking the burndown chart like a hawk, and certifications like CSM.
+
+We should rename that to Agile mentor/coach, even if we want someone to teach Agile principles.
+
+##### Q: Should we hire an Agile coach/mentor?
+
+If you observe that the team is not doing what they need to do or doing what they shouldn't be doing, then
+an Agile coach can help streamline that. Also, in this case, don't make the Agile coach your process.
+
+##### Q: What should be the Agile coach's role?
+
+They teach the team to follow the practice and create other Agile coaches.
+Remember, Agile coaching is not a permanent process.
+It is just an initial push.
+These are just the people who know how to conduct the meeting effectively.
+
+##### Q: What is a facilitator?
+
+After the Agile coach/mentor leaves the team, one person from the team takes up the role of facilitator,
+almost like an Agile coach/mentor but internal.
+They initiate and make sure of a smooth exchange of baton. Don't make a single person facilitator; always rotate them.
+
+You don't need a facilitator if you think your team is mature enough to handle things.
+
+##### Q: What order?
+
+Choose whatever suits you.
+
+Some like the alphabetical order, some like the current position they are sitting in, some like facilitator assigning,
+some like the last person nominating the next person.
+
+You can be creative as soon as meetings are short and effective.
+
+##### Q: Do I need a burn down chart? Or any chart? any board?
```

Comment:
Reading about goals it reminds me that each iteration should have single goal. So where you have three bullets about the agenda. We can rewrite it like
- What did I do towards the iteration goal
- What is inhibiting me from achieving my goals
- What will I do next